phpstan/config.level1.neon

```
@@ -0,0 +1,9 @@
+parameters:
```

This file is currently not used.

Can we remove it?

---

i just moved ignore errors from used config there, so we can in future use it easily

---

What do you think about using `phpstan`'s baseline feature instead?

For reference, see https://medium.com/@ondrejmirtes/phpstans-baseline-feature-lets-you-hold-new-code-to-a-higher-standard-e77d815a5dff.

---

It is good if you want to write better/type-safe code. If you don't it will only bother you.

---

It's an interesting feature IMO if it allows  raising the level faster, but that's something I'd rather investigate later as we get closer to 2.0 release to avoid too many conflicts from 1.x merges. For now let's focus on getting this PR done as is I'd say.

---

I'd also remove this unused file. It will most likely get obsolete before it gets used. And we an easily generate it with the basline feature.